be/src/runtime/data_stream_sender.h

```diff
@@ -147,6 +147,8 @@ class DataStreamSender final : public DataSink {
     std::vector<ExprContext*> _partition_expr_ctxs; // compute per-row partition values
 
     std::vector<Channel*> _channels;
+    // channel send order
```

Comment:
Please give more detailed comments.
Why needs it? And how it works?

Reply:
all right